src/cli.zig

```diff
@@ -188,18 +196,28 @@ pub fn parse_args(allocator: std.mem.Allocator) !Command {
             };
         },
         .start => {
-            if (cluster != null) fatal("--cluster: supported only by 'format' command", .{});
-            if (replica != null) fatal("--replica: supported only by 'format' command", .{});
-            if (verbose != null) fatal("--verbose: supported only by 'version' command", .{});
-
             return Command{
                 .start = .{
                     .args_allocated = args_allocated,
                     .addresses = parse_addresses(
                         allocator,
                         addresses orelse fatal("required: --addresses", .{}),
                     ),
-                    .memory = if (memory) |m| parse_size(m) else config.memory_size_max_default,
+                    .cache_accounts = parse_size_to_count(
+                        tigerbeetle.Account,
+                        cache_accounts,
+                        config.cache_accounts_max,
+                    ),
+                    .cache_transfers = parse_size_to_count(
+                        tigerbeetle.Transfer,
+                        cache_transfers,
+                        config.cache_transfers_max,
+                    ),
+                    .cache_transfers_posted = parse_size_to_count(
+                        tigerbeetle.Transfer,
```

Comment:
This should be `@sizeOf(u256)` (with a TODO to use the actual value type once it's exposed; probably as part of https://github.com/tigerbeetledb/tigerbeetle/issues/264). This cache is for the "posted groove" — it doesn't store transfers directly, just references to them and a little state:


https://github.com/tigerbeetledb/tigerbeetle/blob/cdde7c7254f033a7fdb458ebf6bd0317798233a7/src/lsm/posted_groove.zig#L25-L32

The posted groove is an [unfortunate special case](https://github.com/tigerbeetledb/tigerbeetle/issues/264) of the `Groove` (a group of LSM trees).